app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  def new
```

Comment:
If all this does is render the default sessions#new template, and this method is empty, then there's no reason to have this here.